replica/table.cc

```diff
@@ -3018,6 +3107,10 @@ compaction_backlog_tracker& compaction_group::get_backlog_tracker() {
     return as_table_state().get_backlog_tracker();
 }
 
+compaction_manager& compaction_group::get_compaction_manager() noexcept {
+    return _t.get_compaction_manager();
+}
+
 compaction::table_state& compaction_group::as_table_state() const noexcept {
     return *_table_state;
 }
```

Comment:
unit test?

Reply:
Yes, I planned to add one so it was listed as a missing item.